packages/http-specs/spec-summary.md

```diff
@@ -2118,143 +2152,143 @@ Expected path: /routes/path/explicit/a
 
 Test label expansion with explode: true when passed an array value.
 Param value: ["a","b"]
-Expected path: /routes/label/explode/array.a.b
+Expected path: /routes/path/label/explode/array.a.b
 
 ### Routes_PathParameters_LabelExpansion_Explode_primitive
 
 - Endpoint: `get /routes/path/label/explode/primitive{.param*}`
 
 Test label expansion with explode: true when passed a primitive value.
 Param value: "a"
-Expected path: /routes/label/explode/primitive.a
+Expected path: /routes/path/label/explode/primitive.a
 
 ### Routes_PathParameters_LabelExpansion_Explode_record
 
 - Endpoint: `get /routes/path/label/explode/record{.param*}`
 
 Test label expansion with explode: true when passed a record value.
 Param value: {a: 1, b: 2}
-Expected path: /routes/label/explode/record.a=1.b=2
+Expected path: /routes/path/label/explode/record.a=1.b=2
 
 ### Routes_PathParameters_LabelExpansion_Standard_array
 
 - Endpoint: `get /routes/path/label/standard/array{.param}`
 
 Test label expansion with explode: false when passed an array value.
 Param value: ["a","b"]
-Expected path: /routes/label/standard/array.a,b
+Expected path: /routes/path/label/standard/array.a,b
 
 ### Routes_PathParameters_LabelExpansion_Standard_primitive
 
 - Endpoint: `get /routes/path/label/standard/primitive{.param}`
 
 Test label expansion with explode: false when passed a primitive value.
 Param value: "a"
-Expected path: /routes/label/standard/primitive.a
+Expected path: /routes/path/label/standard/primitive.a
 
 ### Routes_PathParameters_LabelExpansion_Standard_record
 
 - Endpoint: `get /routes/path/label/standard/record{.param}`
 
 Test label expansion with explode: false when passed a record value.
 Param value: {a: 1, b: 2}
-Expected path: /routes/label/standard/record.a,1,b,2
+Expected path: /routes/path/label/standard/record.a,1,b,2
 
 ### Routes_PathParameters_MatrixExpansion_Explode_array
 
 - Endpoint: `get /routes/path/matrix/explode/array{;param*}`
 
 Test matrix expansion with explode: true when passed an array value.
 Param value: ["a","b"]
-Expected path: /routes/matrix/explode/array;a.b
```

Comment:
These changes is due to https://github.com/microsoft/typespec/pull/5717/files

Again, the PR modified the case but didn't synced the summary.